src/Components/WebAssembly/Sdk/src/Microsoft.NET.Sdk.BlazorWebAssembly.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>MSBuild support for building Blazor WebAssembly apps.</Description>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);net46</TargetFrameworks>
+
+    <TargetName>Microsoft.NET.Sdk.BlazorWebAssembly.Tasks</TargetName>
+    <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
+    <Serviceable>true</Serviceable>
+    <SdkOutputPath>$(ArtifactsBinDir)Microsoft.NET.Sdk.BlazorWebAssembly\$(Configuration)\sdk-output\</SdkOutputPath>
+
+    <!-- Allow assemblies outside of lib in the package -->
+    <NoWarn>$(NoWarn);NU5100</NoWarn>
+    <!-- Need to build this project in source build -->
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.Build.Framework" />
+    <Reference Include="Microsoft.Build.Utilities.Core" />
+
+    <ProjectReference
+      Include="..\tools\Microsoft.NET.Sdk.BlazorWebAssembly.Tools.csproj"
+      Targets="Publish"
+      ReferenceOutputAssembly="false"
+      IsImplicityDefined="false"
+      SkipGetTargetFrameworkProperties="true"
+      UndefineProperties="TargetFramework;TargetFrameworks;RuntimeIdentifier;PublishDir" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="_._" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+
+  <Target Name="LayoutDependencies" BeforeTargets="Build"
+    Condition="'$(IsInnerBuild)' != 'true' AND '$(NoBuild)' != 'true'">
```

Comment:
What does `IsInnerBuild` mean here?

Reply:
In a project that multi-targets (`<TargetFrameworks>$(DefaultNetCoreTargetFramework);net46</TargetFrameworks>`), `Build` is called n + 1 times over - the outer build without a TargetFramework, and once for each of the n TFMs in the project. We're avoiding doing work during the n (in this case 2 times) during the inner build.